### ISSUE TITLE
Enhancement: Summary statistics tables seggregated

### ISSUE THREAD

**Issue by Gero1999** [pharmaverse/aNCA#114]
### Description
It would be good to allow the user to obtain another summary statistic table that includes all patients that have been manually excluded from the analysis. In the same way, it would be also positive if for each of the table variables the user can also select the subjects, races or whatever variable values wants to particularly check. 

### Definition of Done
The requirements for the feature to be complete:
- [ ] Include another summary table with only the patients excluded
- [ ] Include an option to filter values of the aggregating variables used in summary tables (i.e, choose only certain USUBJIDs)

**Comment by billdenney** [comment]
Related to #125

FYI, PKNCA has robust data exclusion functionality built-in (see https://billdenney.github.io/pknca/articles/v07-post-processing.html#modifying-results). Summarizing excluded individuals could be added to the PKNCA package without much effort, if helpful. Then, you could use the `summary()` function on a PKNCAresults object for what I think is being requested here.

**Comment by Gero1999** [comment]
It is good to know that you think it could be easily linked. We were not sure about this and our current summary is a customized function. We will then rethink about this!

**Comment by billdenney** [comment]
Yes, if the summary coming out of PKNCA's `summary()` function doesn't fit all of your needs, we should discuss what is missing, and it may be a good fit to add within PKNCA rather than here. It does a lot of work to give useful summaries, already.
